MAINTAINERS-GUIDELINES.md

```diff
@@ -0,0 +1,73 @@
+# Maintainers Guidelines
+
+Maintainership-related decisions must be taken with respect to the rules established in our [governance](GOVERNANCE.md#maintainership). This document provides guidelines for the implementation of these decisions.
+
+[Reviewers](GOVERNANCE.md#reviewers) have no maintainers power, but behave similarly and therefore this document include also guidelines for reviewers. Although they can be added and removed at the sole discretion of the maintainers, it is recommended to use similar criteria when doing so.
```

Comment:
This seems to conflict with the guidelines below which have some rules for when Reviewers are removed.  Perhaps adjust wording?

Reply:
I removed the last sentence from here and rephrased the "Offboarding a Reviewer" section. It should be ok now. Let me know if it works for you :pray: